packages/flutter/lib/src/widgets/animated_switcher.dart

```diff
@@ -240,6 +240,7 @@ class AnimatedSwitcher extends StatefulWidget {
       children: <Widget>[
         ...previousChildren,
         if (currentChild != null) currentChild,
+        if (previousChildren.isEmpty && currentChild == null) const SizedBox.shrink(),
```

Comment:
Do we need to include the Stack in this case at all?

Reply:
Actually, this looks more like a bug in the Stack class? Having a Stack with an empty child list should be totally fine. If that doesn't work there, we should fix that in Stack.

Reply:
@goderbauer
Yes the root cause is in Stack class. I first was too cautious to update RenderStack and first filed another PR that updated the error message.
After having a fresh look, I updated this other PR to replace the assertion, thank you to have a look at it :
https://github.com/flutter/flutter/pull/102642